driver/sdk.md

```diff
@@ -76,78 +84,269 @@ $ bblfsh-sdk update
 managed file "README.md" has changed, discarding changes
 ```
 
-## Creating the AST parser
+`bblfsh-sdk` doesn't update the SDK itself so if you want to use a new version of 
+the SDK you'll also have to update the Go package **first** with:
+
+```bash
+go get -u github.com/bblfsh/sdk/...
+```
+
+If the update gives you some problem you can try to delete the
```

Comment:
Both are fine (checked with a native) just like "algún problema" and "cualquier problema" in spanish, but *any* is more common in this sentence, so fixed.